openml-datarobot/src/test/java/com/feedzai/openml/datarobot/ClassificationBinaryDataRobotModelTest.java

```diff
@@ -0,0 +1,48 @@
+package com.feedzai.openml.datarobot;
+
+import com.feedzai.openml.data.schema.DatasetSchema;
+import com.feedzai.openml.mocks.MockInstance;
+import com.feedzai.openml.provider.exception.ModelLoadingException;
+import com.google.common.primitives.Doubles;
+import org.junit.Test;
+
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests for classifying DataRobot Model with {@link ClassificationBinaryDataRobotModel}.
+ *
+ * @author Sheng Wang (sheng.wang@feedzai.com)
+ * @since 1.0.6
+ */
+public class ClassificationBinaryDataRobotModelTest extends DataRobotModelProviderLoadTest {
+
+    /**
+     * Checks the method ClassificationBinaryDataRobotModel#classify() if it ensures
+     * that DataRobot classifies correctly the index of maximum value of the scores' list.
+     *
+     * @throws ModelLoadingException If the model cannot be loaded.
+     *
+     * @since 1.0.6
+     */
+    @Test
+    public final void testClassify() throws ModelLoadingException {
```

Comment:
As discussed offline, the most generic approach is to add this to AbstractProviderModelBaseTest (in the openml-api repo) so that all providers can (and are effectively forced to) have a regression test for this, because they all implement a test that extends that class.

Check implementations of that AbstractProviderModelBaseTest to understand how it is used and how you could get this method there.

Then work on a pull request on the openml-api repo, and after we're happy with it and you release it, you can come back here, bump the openml-api version, and improve this pull request to automatically cover the regression.